IPython/core/usage.py

```diff
@@ -93,42 +91,39 @@
 MAIN FEATURES
 -------------
 
-* Access to the standard Python help. As of Python 2.1, a help system is
-  available with access to object docstrings and the Python manuals. Simply
-  type 'help' (no quotes) to access it.
+* Access to the standard Python help with object docstrings and the Python
+  manuals. Simply type 'help' (no quotes) to invoke it.
 
 * Magic commands: type %magic for information on the magic subsystem.
 
 * System command aliases, via the %alias command or the configuration file(s).
 
 * Dynamic object information:
 
-  Typing ?word or word? prints detailed information about an object.  If
-  certain strings in the object are too long (docstrings, code, etc.) they get
-  snipped in the center for brevity.
+  Typing ?word or word? prints detailed information about an object.  Long
+  strings (docstrings, code, etc.) get snipped in the center for brevity.
```

Comment:
Reading file into string and showing it with `word?` produced snipped output in "String form:" field. `docstring` was paged.